calc.rb

```diff
@@ -0,0 +1,74 @@
+class Object
+  def is_number?
+    self.to_f.to_s == self.to_s || self.to_i.to_s == self.to_s
+  end
+end
+
+puts "Welcome to the okayest calculator ever!"
+
+puts "Are we using floats or integers today?"
+num_class = gets.chomp.downcase
+
+num_options = ["integer", "integers", "float", "floats"]
+
+until num_options.include?(num_class)
+  print "Was that a float or integer? "
+  num_class = gets.chomp.downcase
+end
+
+print "Please enter an operation: "
+user_operation = gets.chomp.downcase
+
+operations = ["add", "+", "subtract", "-", "multiply", "*", "divide", "/",
+"^", "exponent", "**", "%", "modulo"]
+
+until operations.include?(user_operation)
+  print "Please enter valid operation, remember-this calculator is only okay: "
+  user_operator = gets.chomp.downcase
```

Comment:
You probably want to assign this value to the variable user_operation. If you put in an erroneous operation and get into this until loop, you end up never being able to escape, hehehe